packages/manager/apps/dedicated/client/app/dedicated/server/bandwidth/public-order/public-order.html

```diff
@@ -1,129 +1,143 @@
-<div
-    data-ng-if="!$ctrl.model.orderUrl"
-    data-wizard
-    data-wizard-on-cancel="$ctrl.cancel.bind($ctrl)"
-    data-wizard-on-finish="$ctrl.order.bind($ctrl)"
-    data-wizard-hide-cancel-button="$ctrl.isLoading"
-    data-wizard-hide-previous-button="$ctrl.isLoading"
-    data-wizard-hide-confirm-button="!$ctrl.isLoading"
-    data-wizard-title=":: 'server_order_bandwidth_title' | translate"
-    data-wizard-bread-crumb
-    data-wizard-confirm-button-text=":: 'wizard_pay' | translate"
+<!-- Data loading modal -->
+<oui-modal
+    data-ng-if="$ctrl.isLoading"
+    data-heading="{{:: 'server_order_bandwidth_title' | translate}}"
 >
+    <div class="text-center">
+        <oui-spinner></oui-spinner>
+        <p data-translate="server_order_bandwidth_load" class="d-block"></p>
```

Comment:
**question:** Is `d-block` still needed? Because `p` is a block element.
```suggestion
        <p data-translate="server_order_bandwidth_load"></p>
```